src/Components/CommentCard.jsx

```diff
@@ -3,44 +3,44 @@ import { useEffect } from "react";
 import { useState } from "react";
 import axios from "axios";
 import { FaTimes } from "react-icons/fa";
+import { fetchReviewComments, postReviewComment } from "../Components/Api";
+import "../StyleSheets/CommentCard.css";
 
 const CommentCard = ({ review_id }) => {
   const [comments, setComments] = useState([]);
   const id = review_id;
   const [err, setErr] = useState(null);
+  const [value, setValue] = useState("");
 
   useEffect(() => {
-    axios
-      .get(`https://games-nc.herokuapp.com/api/reviews/${id}/comments`)
-      .then((res) => {
-        setComments(res.data.comments);
-      });
-  }, [id]);
+    fetchReviewComments(id).then((data) => {
+      setComments(data.comments);
+    });
+  }, [comments.length, id]);
 
   const handleSubmit = (e) => {
     e.preventDefault();
-    let comment = e.target.firstElementChild.value;
+    let body = e.target.firstElementChild.value;
+    let username = "tickle122";
     let commentObj = {
-      body: comment,
+      body: "uploading comment",
       comment_id: Math.random(),
```

Comment:
27 - we don't want to be giving our comments comment_id.  The back end automatically assigns comment_id to every comment (serial primary key).

Sometimes when I delete comments I have just made I am getting a 404 error message which I think is related to this.  I will send you a picture of the error